etc/enigma.appdata.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2014 Ryan Lerch <rlerch@redhat.com>, Andreas Lochmann <and.lochmann@googlemail.com> -->
-<application>
-  <id type="desktop">enigma.desktop</id>
+<component type="desktop-application">
+  <id>org.nongnu.enigma</id>
```

Comment:
We have an own domain name `enigma-game.org`. I'd prefer to use this. (Even if it just redirects to nongnu for now.)

Reply:
Changing the ID has some consequences though:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#renaming-id-tag